lib/pages/portal/service/website_provider.dart

```diff
@@ -220,4 +248,26 @@ class WebsiteProvider with ChangeNotifier {
       return false;
     }
   }
+
+  void updateVisits(Website website) async {
```

Comment:
A name like `incrementNumberOfVisits` would be more specific in showing what this method does.